docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -117,41 +133,90 @@ coordinate is unevenly spaced in altitude:
-the new data in a plot: 
+Let's look at the original data, the interpolation line and
+the new data in a plot. This will help us to see what is going on:
 
 .. plot:: userguide/regridding_plots/interpolate_column.py
 
-As we can see with the red diamonds on the extremes of the altitude values, we have
+The red diamonds on the extremes of the altitude values show that we have
 extrapolated data beyond the range of the original data. In some cases this is desirable
-functionality, and for others it is not - for instance, this column defined
-a surface altitude value of 414m, so extrapolating "air potential temperature" at 400m
-in this case makes little physical sense.
-
-Fortunately we can control the extrapolation mode when defining the interpolation scheme
-with the ``extrapolation_mode`` keyword.  For :class:`iris.analysis.Linear` the
-``extrapolation_mode`` must be one of ``linear``, ``error``, ``nan``, ``mask`` or
-``nanmask``. To mask the values which lie beyond the range of the original data, using
-the ``mask`` extrapolation mode is just a matter of constructing the appropriate scheme
-and passing it through to the :meth:`~iris.cube.Cube.interpolate` method:
+functionality but in other cases it is not. For example, this column defines
+a surface altitude value of 414m, so extrapolating an "air potential temperature" at 400m
+makes little physical sense in this case.
+
+We can control the extrapolation mode when defining the interpolation scheme.
+Controlling the extrapolation mode means we can avoid situations like the above where
+extrapolating values makes little physical sense.
+
+The extrapolation mode is controlled by the ``extrapolation_mode`` keyword.
+For the available interpolation schemes available in Iris, the ``extrapolation_mode``
+keyword must be one of:
+ * ``extrapolate`` (the extrapolation points will be calculated by extending
+    the gradient of the closest two points),
+ * ``error`` (a ValueError exception will be raised, notifying an attempt to extrapolate),
+ * ``nan`` (the extrapolation points will be be set to NaN),
+ * ``mask`` (the extrapolation points will always be masked, even if the source
+    data is not a MaskedArray), or
+ * ``nanmask`` (if the source data is a MaskedArray the extrapolation points
+    will be masked. Otherwise they will be set to NaN).
+
+Using an extrapolation mode is achieved by constructing an interpolation scheme
+with the extrapolation mode keyword set as required. The constructed scheme
+is then passed to the :meth:`~iris.cube.Cube.interpolate` method.
+For example, to mask values that lie beyond the range of the original data:
 
     >>> scheme = iris.analysis.Linear(extrapolation_mode='mask')
     >>> new_column = column.interpolate(sample_points, scheme)
+    >>> print new_column.coord('altitude').points
+    [           nan   494.44451904   588.88891602   683.33325195   777.77783203 \
+   872.222229     966.66674805  1061.11108398  1155.55541992            nan]
```

Comment:
Hmmm do you need to indent it ? ... can't you find an example from elsewhere in the docs? We must have done that somewhere else, surely ...